components/Homepage.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react'
```

Comment:
- ```HomePage``` is a **page**, it's not a component, you should define ```home.tsx``` page in this path: ```pages/```. Remove this file from ```components/``` directory.